lib/routes/account.js

```diff
@@ -998,7 +962,7 @@ module.exports = function (
             lastAccessTime: isA.number().min(0).required().allow(null),
             // We previously allowed devices to register with arbitrry unicode names,
             // so we can't assert DISPLAY_SAFE_UNICODE in the response schema.
-            name: isA.string().max(255).required(),
+            name: isA.string().max(255).required().allow('').allow(null),
```

Comment:
Very good point. Have swapped `required` for `optional`, plus it seems like I can ditch the `allow(null)` too, although maybe that is preferred to be explicit?